serverless.yml

```diff
@@ -46,6 +46,7 @@ provider:
         - { "Fn::GetAtt": ["StreetArtDynamoDBTable", "Arn" ] }
   environment:
     ART_TABLE: ${self:custom.streetArtTableName}
+    MONGO_DB: mongodb+srv://streettagged:XrdG9JQShFmAy4rd@streettagged-z3s5t.mongodb.net/streettagged?retryWrites=true&w=majority
```

Comment:
these look like secrets? or no? this is apublic repo

Reply:
We have have some where to store it